README.md

```diff
@@ -413,7 +414,7 @@ bin/copyfromcontainer --all
 
 Running Docker on Linux should be pretty straight-forward. Note that you need to run some [post install commands](https://docs.docker.com/install/linux/linux-postinstall/) as well as [installing Docker Compose](https://docs.docker.com/compose/install/) before continuing. These steps are taken care of automatically with Docker Desktop, but not on Linux.
 
-Copy `docker-compose.dev-linux.yml` to `docker-compose.dev.yml` before installing Magento to take advantage of this setup.
+Edit `bin/docker-compose` to add `-f docker-compose.dev-linux.yml` with `${DOCKER_COMPOSE}` before installing Magento to take advantage of this setup.
```

Comment:
Same here, need to keep the "Copy `docker-compose.dev-linux.yml` to `docker-compose.dev.yml`" verbiage at the beginning of this paragraph, since `docker-compose.dev-linux.yml` doesn't yet exist.